.github/workflows/beam_LoadTests_Go_CoGBK_Flink_batch.yml

```diff
@@ -0,0 +1,128 @@
+      github.event.comment.body == 'Run Load Tests Go CoGBK Flink Batch'
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 720
+    name: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+    strategy:
+      matrix:
+        job_name: ["beam_LoadTests_Go_CoGBK_Flink_Batch"]
+        job_phrase: ["Run Load Tests Go CoGBK Flink Batch"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
+      - name: Prepare test arguments
+        uses: ./.github/actions/test-arguments-action
+        with:
+          test-type: load
+          test-language: go
+          argument-file-paths: |
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_CoGBK_Go_Batch_Flink_MultipleKey.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_CoGBK_Go_Batch_Flink_Reiteration_10KB.txt
+            ${{ github.workspace }}/.github/workflows/load-tests-job-configs/config_CoGBK_Go_Batch_Flink_Reiteration_2MB.txt
+          arguments: |
+            --job_name=load-tests-go-flink-batch-cogbk-$(date '+%m%d%H%M%S' --utc)
+      - name: Start Flink with parallelism 5
+        env:
+          FLINK_NUM_WORKERS: 5
+        run: |
+          cd ./.test-infra/dataproc; ./flink_cluster.sh create
+      - name: run CoGBK Flink Batch Go Load Test 1 (multiple keys)
+        timeout-minutes: 240
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=cogbk \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Go_CoGBK_Flink_Batch_test_arguments_1 }}' \
+      - name: run CoGBK Flink Batch Go Load Test 2 (reiterate 4 times 10KB values)
+        timeout-minutes: 240
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=cogbk \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Go_CoGBK_Flink_Batch_test_arguments_2 }}' \
+      - name: run CoGBK Flink Batch Go Load Test 3 (reiterate 4 times 2MB values)
+        timeout-minutes: 240
+        uses: ./.github/actions/gradle-command-self-hosted-action
+        with:
+          gradle-command: :sdks:go:test:load:run
+          arguments: |
+            -PloadTest.mainClass=cogbk \
+            -Prunner=FlinkRunner \
+            '-PloadTest.args=${{ env.beam_LoadTests_Go_CoGBK_Flink_Batch_test_arguments_3 }}'
+      - name: Teardown Flink
+        if: always()
+        run: |
+          ./.test-infra/dataproc/flink_cluster.sh delete
```

Comment:
let's add the `${{ github.workspace }}` at the begining in order to have an absolute path

Reply:
Done